micronota/bfillings/_base.py

```diff
@@ -0,0 +1,83 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015--, micronota development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from abc import ABCMeta, abstractmethod
+
+from skbio import Sequence
+
+
+class IntervalMetadataPred(metaclass=ABCMeta):
```

Comment:
Do we need two classes?  That is already taken care of with straight-up `skbio.Sequences`.